app/controllers/users_controller.rb

```diff
@@ -31,6 +40,13 @@ def add_contact
 
   private
 
+  def handle_error
+      messages = @user.errors.full_messages
+      error_heading = I18n.t 'errors.messages.not_saved.other', count: messages.count, resource: SocialAccount
+      log = {heading: error_heading, messages: messages}
+      flash[:danger] = log
```

Comment:
I would display the error messages in the form, is there a reason why they are displayed as a flash message?

Reply:
I assume that all error messages consists of a heading and all messages that belong to the error. It makes things a little bit easier and more generic to display all parts of the error message as a flash message.